torch/onnx/_internal/fx/op_validation.py

```diff
@@ -267,10 +241,103 @@ def _fx_args_to_torch_args(
 def wrap_fx_args_as_torch_args(
     complete_args: List[fx_type_utils.Argument],
     complete_kwargs: Dict[str, fx_type_utils.Argument],
-) -> Tuple[tuple, dict]:
+) -> Tuple[List[fx_type_utils.Argument], Dict[str, fx_type_utils.Argument]]:
     """Prepare torch format args and kwargs for op-level validation by using fake tensor to create real tensor to feed in ops"""
 
     # NOTE: This function only supports FakeTensor with concrete shapes
     torch_args: List[fx_type_utils.Argument] = _fx_args_to_torch_args(complete_args)
     torch_kwargs = complete_kwargs
-    return tuple(torch_args), torch_kwargs
+    return torch_args, torch_kwargs
+
+
+# NOTE: Referenced from onnxscript internal function: _tag_arguments_with_param_schemas.
+# Importing this function makes the code less robust, as it is not a public API on onnxscript.
+@_beartype.beartype
+def _split_tangled_arguments_with_param_schemas(
```

Comment:
synced offline, some clarification and renaming on terminologies is needed

Reply:
Use `_convert_torch_args_to_onnxfunction_args`